userspace/plugin/plugin_types.h

```diff
@@ -279,6 +279,20 @@ typedef void ss_plugin_t;
 //
 typedef void ss_instance_t;
 
+//
+// Libsinsp logger severities
```

Comment:
```suggestion
// Severity available in the logging facility provided by the framework
```
We try to not mention libscap and libsinsp in the plugin API to make it as agnostic as possible.